projects/batfish-common-protocol/src/org/batfish/datamodel/Configuration.java

```diff
@@ -72,6 +72,10 @@
 
    private static final String TACACS_SOURCE_INTERFACE_VAR = "tacacsSourceInterface";
 
+   private static final int VLAN_NORMAL_MIN_DEFAULT = 1;
```

Comment:
Yes, these are common across all vendors, and Cisco has further refinements onto them (VLAN id is 12-bit long per 802.11Q, with 0 and 0xFFF not usable). Cisco specific rules, see for example: http://www.cisco.com/c/en/us/td/docs/switches/datacenter/nexus5000/sw/configuration/guide/cli/CLIConfigurationGuide/VLANs.html